### PR TITLE
LPS-57308 Metal.js Bundle

### PR DESCRIPTION
**Pull request by jbalsas**
/cc @eduardolundgren @natecavanaugh @ipeychev

PS: I've tested the build, and it works in OS X, Windows 7 and the CI servers

### CODE REVIEW

File: modules/frontend/frontend-js-web/src/META-INF/resources/liferay/modules.js [hunk 1]
```diff
@@ -946,5 +946,14 @@
 		useBrowserConsole: false
 	};
 
+	Loader.addModule(
```
Comment (jbalsas) [inline modules/frontend/frontend-js-web/src/META-INF/resources/liferay/modules.js]: Hey guys, I saw we have a soyutils copy inside the `frontend-js-web` module. This is currently being used mainly by `ddm`.

I've reused that one for now, but I think we could create a different bundle and grab the file from `bower` as we do here and in the `babel-polyfill` module. What do you think?

**Comment by brianchandotcom** [comment]
@jbalsas will these files be served directly through the web?

you named the file *-web, but bnd file doesn't have a:

Web-ContextPath: /frontend-css-web

See frontend-css-web, and compare that to frontend-common-css.

If it doesn't serve a resource that requires a web url, then please remove the web from the dir.

Please resend. Thx.

**Comment by jbalsas** [comment]
Hi Brian

Yes, the files will be served via web. The Web-ContextPath property is automatically generated in build time by the BowerAnalyzerPlugin based on the contents of the bower.json file.

I'm afk, cand you reopen? Otherwise I'll resend later tonight.

**Comment by jbalsas** [comment]
Resent https://github.com/brianchandotcom/liferay-portal/pull/29685

**Pull request closed without merging**